akka-bbb-apps/src/main/scala/org/bigbluebutton/core/apps/breakout/CreateBreakoutRoomsCmdMsgHdlr.scala

```diff
@@ -79,6 +81,12 @@ trait CreateBreakoutRoomsCmdMsgHdlr extends RightsManagementTrait {
     for (breakout <- rooms.values.toVector) {
       val roomSlides = if (breakout.allPages) -1 else presSlide;
 
+      val filteredPluginProp = liveMeeting.props.pluginProp.asScala
+        .filter { case (key, _) =>
+          getPlugins(liveMeeting.plugins).get(key).exists(_.manifest.content.enabledForBreakoutRooms)
+        }
+        .asJava
```

Comment:
Move this section outside the for `(room <- msg.body.rooms)` loop. Since all rooms will receive the same value, running the logic for each room is inefficient.